### DISCUSSION TITLE
Global Flags like --cache-dir

### DISCUSSION THREAD

**Discussion by FlorianEndel** [cupcakearmy/autorestic#109] [Q&A] [answered]
How can [global flags](https://restic.readthedocs.io/en/latest/manual_rest.html) (like `--cache-dir directory` ) be included in the yaml-config?

**Comment by cupcakearmy** [comment]
For now your could use https://autorestic.vercel.app/location/options to enable it. However they would be limited to a single command. Maybe adding something like

```yaml
options:
  global:
    cache-dir: ./some/dir
```

Could be the way to go. What do you think? Feel free to open a ticket btw

**Reply by cupcakearmy** [reply]
I've done a ticket, it's a good suggestions https://github.com/cupcakearmy/autorestic/issues/115

**Comment by cupcakearmy** [comment, marked as answer]
Implemented btw https://autorestic.vercel.app/location/options#global-options
